README.md

```diff
@@ -1,10 +1,43 @@
 Tensor Compute Primitives
 =========================
 
-Mid level intermediate representation for machine learning programs.
+Mid-level intermediate representation for machine learning programs.
 
-If you have `clang` installed you can build & run all tests using
+![Bazel Build](https://github.com/cruise-automation/mlir-tcp/actions/workflows/bazelBuildAndTestTcp.yml/badge.svg)
 
+## Project Communication
+
+- For general discussion use `#mlir-tcp` channel on the [LLVM Discord](https://discord.gg/xS7Z362)
+- File [issues](https://github.com/cruise-automation/mlir-tcp/issues) on GitHub
+
+## Developer Guide
+
+To build TCP using Bazel, follow these steps:
+
+1. For a quick start, launch an interactive docker container with clang (and lld) pre-installed:
+
+```shell
+./docker/run_docker.sh
```

Comment:
Do we really need this for TCP? Things were more complicated for Torch-MLIR, but for TCP we just need clang.

Reply:
You're correct that this is an optional step (marked it as `Optional` now). I'd recommended it because we use the same minimal docker to source the GHA runners for CI (for reproducibility reasons and cruise restrictions on installing stuff on GHA runners that are not from GitHub marketplace) and historically (with Torch-MLIR) we've seen issues where CI failures were hard to reproduce locally for our users (due to environment issues, clang versions). Having a docker enforces some form of hermeticity. Another advantage is in case we grow to take more dependencies this is more future proof.